### PR TITLE
Input component: PR

### PR DESCRIPTION
**Pull request by AntonPDC**
@ZacharyRizer Looks like it works. Probably not the prettiest way of doing it

### CODE REVIEW

File: src/TodoList.tsx [hunk 1]
```diff
@@ -43,11 +48,16 @@ export const TodoList: React.FC = () => {
...
         ))}
       </ul>
-      <input
+      <Input
+        id={Date.now()}
+        placeholder="Enter Todo Item"
+        onChange={handleInputChange}
+      />
+      {/* <input
         type="text"
```
Comment (ZacharyRizer) [inline src/TodoList.tsx:57]: We need a prop to represent the `type` parameter on a standard HTML `<input>`. This is the prop that determines if the input is text, date, etc. Check out the documentation here: [input docs](https://www.w3schools.com/tags/tag_input.asp)

File: src/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import "./Input.css";
+
+interface InputProps {
+  id: number;
```
Comment (ZacharyRizer) [inline src/components/Input/Input.tsx:4]: What is the purpose of the `id` property here? What is its function?

File: src/components/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  placeholder: string;
+  onChange: (value: string) => void;
+}
+
+function Input({ id, placeholder, onChange }: InputProps) {
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (ZacharyRizer) [inline src/components/Input/Input.tsx:10]: Great job on figuring this part out. You could simplify this by mimicking the way that it was done on the basic `<input>` before. Like this: `onChange={(e) => setTodoText(e.currentTarget.value)}`, except you would replace the setTodoText with your prop onChange.

File: src/TodoList.tsx [hunk 4]
```diff
@@ -43,11 +48,16 @@ export const TodoList: React.FC = () => {
           </li>
         ))}
       </ul>
-      <input
+      <Input
+        id={Date.now()}
+        placeholder="Enter Todo Item"
+        onChange={handleInputChange}
+      />
+      {/* <input
```
Comment (ZacharyRizer) [inline src/TodoList.tsx:56]: Remove this commented code.

**Review by ZacharyRizer — CHANGES_REQUESTED** [review]
Added comments.

File: src/TodoList.tsx (hunk 4 above)
Comment (AntonPDC) [reply src/TodoList.tsx]: forgot to save

File: src/components/Input/Input.tsx (hunk 2 above)
Comment (AntonPDC) [reply src/components/Input/Input.tsx]: Removed ID from prop

**Pull request merged**